knowledge-base/datagrid-cell-doubletap.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Execute CellTap command on CellDoubleTap in DataGrid
+description: Execute the CellTap command in CellDoubleTap command in the RadDataGrid for .NET MAUI.
+type: how-to
+page_title: Execute the CellTap command in CellDoubleTap command in DataGrid
+slug: datagrid-cell-doubletap
+position: 
+tags: 
+ticketid: 1611664
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>5.2.0</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>DataGrid for MAUI</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This how-to article describes how to execute the `CellTap` command in `CellDoubleTap` command in the RadDataGrid for .NET MAUI.
+
+## Solution
+
+Here is an example how to execute the `CellTap` command in `CellDoubleTap` command.
+
+1. Create the needed business objects, for example, type Country with the following properties:
+
+```C#
```

Comment:
Indent all code blocks so that each block is rendered as nested under the numbered item where it belongs. The left side shows how the article is rendered now, without indentation of the code blocks, and the right side shows how it looks with proper indentation.
![image](https://github.com/telerik/maui-docs/assets/60105689/3a06441e-b22d-4795-ae4e-c034220f63e0)